best-practices/README.md

```diff
@@ -189,10 +189,9 @@ Email
 
 JavaScript
 ----------
-* Use Coffeescript, ES6 with [babel], or another language that compiles to
-  JavaScript
+* Use JavaScript with [babel], or another language that compiles to JavaScript.
```

Comment:
@teoljungberg great point; I think my concern here was that it could be construed that we're encouraging vanilla JS/ES5, but if it seems clear that we should be using babel for transpiling whatever the latest version is, I'm good with this! :+1: